rules/windows/process_creation/proc_creation_win_findstr_lnk.yml

```diff
@@ -24,7 +24,8 @@ detection:
               - 'FIND.EXE'
               - 'FINDSTR.EXE'
     selection_cli:
-        CommandLine|endswith: '.lnk'
+        - CommandLine|endswith: '.lnk'
+        - CommandLine|endswith: '.lnk"'
```

Comment:
```suggestion
        CommandLine|endswith:
            - '.lnk'
            - '.lnk"'
```

Reply:
Also @meiliumeiliu please update the modified date.

Also please unprotect your branch when opening a PR to speed up the review process :)

Reply:
And as another side note when opening a PR it states in the PR body to not remove anything. Yet you deleted everything including the SigmaHQ convention and changelog that you're required to read and fill :)